Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM python:2.7-slim
+
+MAINTAINER Azavea
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    g++ \
+    git \
+    gfortran \
+    libatlas-base-dev \
+    libblas-dev \
+    libgeos-dev \
+    liblapack-dev \
+    libopenmpi-dev \
+    libtool \
+    openmpi-bin \
+    python-all-dev \
+    python-pip \
+    wget
+
+# Download, build, and install GDAL
+RUN cd tmp && \
+    wget http://download.osgeo.org/gdal/gdal-1.9.0.tar.gz && \
```

Comment:
You can avoid writing the archive file to disk by making `wget` output to `stdout` and then piping that to `tar` with something like:

```bash
$ wget -qO- http://www.scala-lang.org/files/archive/scala-${SCALA_VERSION}.tgz \
  | tar -xzC ${SCALA_HOME} --strip-components=1
```